utils/update-checkout

```diff
@@ -34,56 +38,53 @@ sys.path.append(os.path.join(SCRIPT_DIR, 'swift_build_support'))
 from swift_build_support import shell  # noqa (E402)
 
 
-def update_single_repository(repo_path, branch, reset_to_remote, should_clean,
-                             cross_repo):
+def update_single_repository(args):
```

Comment:
That is correct, but you can no longer call `update_single_repository` supplying just those 5 arguments, you'd have to wrap them in either tuple or a list. I just pointed it out as it might cause problems.